code/restful-api/restful-api/Models/Sorteio.cs

```diff
@@ -0,0 +1,53 @@
+using System;
+
+namespace RestfulApi.Models
+{
+    public class Sorteio
+    {
+        public long Id { get; set; }
+
+        public DateTime DataFim { get; set; }
+
+        public string Nome { get; set; }
+
+        public string descricao { get; set; }
+
+        public DateTime DataInicio { get; set; }
+
+        public DateTime HorarioInicio { get; set; }
+
+        public DateTime HorarioFim { get; set; }
+
+        public string Premios { get; set; }
+
+        public string Observacao { get; set; }
+
+        public long OrganizadorId { get; set; }
+
+        public Organizador Organizador { get; set; }
+
+        public Usuario[] Usuario { get; set; }
```

Comment:
Did you check if this is the proper format for a list of `Usuario`? I'm not sure.

Reply:
Criado a class SorteioGanhadores para realizar relacionamento n-n. Retirado usuarios.

Reply:
👍